src/viz-parole-population/VizParolePopulation.js

```diff
@@ -21,6 +46,100 @@ const MapCaption = styled.figcaption`
   font: ${(props) => props.theme.fonts.body};
 `;
 
+const ParoleDemographicsWrapper = styled.figure`
+  height: 64px;
+  width: 376px;
+`;
+
+const ParoleDistrictCountWrapper = styled.div`
+  text-align: right;
+`;
+
+function VizParaoleDistrictCount({ data, currentDistrict }) {
+  if (!currentDistrict) return null;
+
+  const districtData = data[currentDistrict];
+
+  // Each individual district has an "aggregate record" which is defined
+  // by each of its demographic entries being set to "ALL".  Unfortunately,
+  // when no district is selected, there is no "aggregate record" so to get
+  // the population total, we sum the total of the population across a
+  // single dimension, in this case GENDER.
+  const genderKeys =
+    currentDistrict === TOTAL_KEY ? Object.keys(GENDERS) : [TOTAL_KEY];
+
+  const count = sum(
+    districtData
+      .filter(
+        (record) =>
+          record.race_or_ethnicity === TOTAL_KEY &&
+          genderKeys.includes(record.gender) &&
+          record.age_bucket === TOTAL_KEY
+      )
+      .map((record) => +record.total_supervision_count)
+  );
+
+  return (
+    <ParoleDistrictCountWrapper>
+      <Statistic value={formatAsNumber(count)} label="People on parole" />
+    </ParoleDistrictCountWrapper>
+  );
+}
+
+VizParaoleDistrictCount.propTypes = {
+  data: PropTypes.objectOf(PropTypes.arrayOf(PropTypes.object)).isRequired,
+  currentDistrict: PropTypes.string,
+};
+
+VizParaoleDistrictCount.defaultProps = {
+  currentDistrict: undefined,
+};
+
+function VizParoleDemographicBarChart({ data, currentDistrict, dimension }) {
+  if (!currentDistrict) return null;
+
+  const districtData = data[currentDistrict];
+  const dimensionKey = dimension.key;
+  const dimensionValues = dimension.values;
+
+  const dimensionData = Object.entries(dimensionValues)
+    .map(([demographic, label]) => {
+      return districtData
+        .filter(
+          (record) =>
+            // i.e. record["race_or_ethnicity"] === "OTHER"
+            record[DATA_KEY_TRANSLATIONS[dimensionKey]] === demographic
+        )
+        .map((record) => ({
+          // i.e. BAR_CHART_VISUALIZATION_COLORS["gender"]["FEMALE"]
+          color: BAR_CHART_VISUALIZATION_COLORS[dimensionKey][demographic],
+          label,
+          value: +record.total_supervision_count,
+        }))
+        .shift();
+    })
+    .filter((record) => record);
+
+  return (
+    <ParoleDemographicsWrapper>
+      <ProportionalBar data={dimensionData} title={DIMENSIONS[dimension.key]} />
```

Comment:
i think we should be including the legends here?

Reply:
I was matching the mocks, which doesn't include the legends, but I agree that they should be included.  I'll need to do some work to get them added though because including them messes up the charts when the legend text wraps.  Example:

<img width="434" alt="Screen Shot 2020-07-13 at 7 04 56 PM" src="https://user-images.githubusercontent.com/25503/87362554-f8236400-c53c-11ea-87dc-9d31698908aa.png">

Reply:
yeah I noticed that ... I didn't really anticipate this when I created that chart component unfortunately ... looks like not-fun to fix! 😎